src/Pact/Analyze/Translate.hs

```diff
@@ -1231,8 +1233,22 @@ translateNode astNode = withAstContext astNode $ case astNode of
     Some ty tm' <- translateNode tm
     pure $ Some SStr $ CoreTerm $ Typeof ty tm'
 
-  AST_Step                -> throwError' $ NoPacts astNode
-  AST_NFun _ "pact-id" [] -> throwError' $ NoPacts astNode
+  AST_Step _node entity exec rollback -> do
+    Some ty exec' <- translateNode exec
+    entity' <- case entity of
+      Nothing -> pure Nothing
+      Just tm -> do
+        Some SStr entity' <- translateNode tm
+        pure $ Just entity'
+    rollback' <- case rollback of
+      Nothing -> pure Nothing
+      Just tm -> Just <$> translateNode tm
+    inStep <- view teInStep
+    let step = Some SStr $ Sequence
+          (Some ty (Step entity' (exec' :< ty) rollback'))
+          (StrLit "step done")
```

Comment:
I guess with `SStr` we are picking an arbitrary type here?

Reply:
Yeah. My understanding is that this value is never used. This is definitely a hack.